README.md

```diff
@@ -60,6 +60,9 @@ Active             | Did the project get any commits in the last 90 days?
 Branch-Protection  | Does the project use [Branch Protection](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/about-protected-branches) ?
 Packaging          | Does the project build and publish official packages from CI/CD, e.g. [GitHub Publishing](https://docs.github.com/en/free-pro-team@latest/actions/guides/about-packaging-with-github-actions#workflows-for-publishing-packages) ?
 Vulnerabilities    | Does the project have unfixed vulnerabilities? Uses the [OSV service](https://osv.dev).
+Token-Permissions  | Does the project declare GitHub workflow tokens as [read only](https://docs.github.com/en/actions/reference/authentication-in-a-workflow)?
+Automatic-Dependency-Update | Does the project use tools to automatically update its dependencies?
+Binary-Artifacts   | Is the project free of checked-in binaries?
```

Comment:
Thanks a ton. Can you sort the list of checks and then just commit the PR.

Reply:
sort alphabetically?

Reply:
Yes, sort alphabetically! I was thinking of doing that. Thanks

Reply:
yes :)